modules/olm-mirroring-catalog.adoc

```diff
@@ -87,6 +87,11 @@ wrote mirroring manifests to manifests-{index-image}-1614211642 <2>
 ----
 <1> Directory for the temporary `index.db` database generated by the command.
 <2> Record the manifests directory name that is generated. This directory name is used in a later step.
++
+[NOTE]
+====
+Red Hat Quay does not support nested repositories. As a result, `oc adm catalog mirror` will fail with a `401` unauthorized error. As a workaround, you can use the `--max-components=2` option when running `oc adm catalog mirror`, which disables the creation of nested repositories. For more information on this workaround, see link:https://access.redhat.com/solutions/5440741[Unauthorized error thrown while using catalog mirror command with Quay registry]. 
```

Comment:
```suggestion
Red Hat Quay does not support nested repositories. As a result, running the `oc adm catalog mirror` command will fail with a `401` unauthorized error. As a workaround, you can use the `--max-components=2` option when running the `oc adm catalog mirror` command to disable the creation of nested repositories. For more information on this workaround, see the link:https://access.redhat.com/solutions/5440741[Unauthorized error thrown while using catalog mirror command with Quay registry] Knowledgebase Solution article. 
```

We usually write the `oc xyz` command, as opposed to just the command itself. The other minor tweaks are up to you whether you find them helpful or not. I dug around to find out how we refer to kbase articles but it's all over the map. Because the link is pointing away from the docs site, it could be helpful to say something like "solution article" or variation thereof.